topics/chip-seq/tutorials/estrogen-receptor-binding-site-identification/tutorial.md

```diff
@@ -0,0 +1,531 @@
+
+> ### {% icon question %} Questions
+>
+> 1. Which information do you find in a SAM/BAM file? What is the additional information compared to a FASTQ file.
+>
+>    <details>
+>    <summary>Click to view answers</summary>
+>    <ol type="1">
+>       <li>Sequences and Quality information, like FASTQ</li>
+>       <li>Mapping information; Location of the read on the chromosome; Mapping quality ...</li>
+>       <li>For more information please see <a href="https://en.wikipedia.org/wiki/SAM_(file_format)">https://en.wikipedia.org/wiki/SAM_(file_format)</a></li>
+>    </ol>
+>    </details>
+{: .question}
+
+# ChIP-seq Quality Control
+
+We already checked the quality of the raw sequencing reads in the first step.
+Now we would like to test the quality of the ChIP-seq preparation, to know if your ChIP-seq samples are more enriched than the control (input) samples.
+
+## Correlation between samples
+
+To assess the similarity between the replicates of the ChIP and the input, respectively, it is a common technique to calculate the correlation of
+read counts on different regions for all different samples.
+We expect that the replicates of the ChIP-seq experiments should be clustered more closely to each other than the replicates of the input sample.
+That is, because the input samples should not have enriched regions included - remember the immuno-precipitation step was skiped during the sample preparation.
+
+To do that, we need at first to catch up for all our samples and re-run the previous steps (quality control and mapping) on each sample.
+To save time, we already did that and we can now work directly on the BAM files of the 8 samples
+
+> ### {% icon hands_on %} Hands-on: Correlation between samples
+>
+> 1. Create a new history
+> 2. Import the 8 BAM files from [Zenodo](https://doi.org/10.5281/zenodo.892432) or from the data library into the history
+>    - [`patient1_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_ChIP_ER_good_outcome.bam)
+>    - [`patient1_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient1_input_good_outcome.bam)
+>    - [`patient2_ChIP_ER_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_ChIP_ER_good_outcome.bam)
+>    - [`patient2_input_good_outcome.bam`](https://zenodo.org/record/892432/files/patient2_input_good_outcome.bam)
+>    - [`patient3_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_ChIP_ER_poor_outcome.bam)
+>    - [`patient3_input_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient3_input_poor_outcome.bam)
+>    - [`patient4_ChIP_ER_poor_outcome.bam`](https://zenodo.org/record/892432/files/patient4_ChIP_ER_poor_outcome.bam)
+>    - [`patient4_input_poor_outcome.bam`](url: https://zenodo.org/record/892432/files/patient4_input_poor_outcome.bam)
+>
+> 3. **multiBamSummary** {% icon tool %} with
```

Comment:
Can you add here some extra info about deepTools in general and then on multiBamSummary? Something along this lines:

3. To compute the correlation between the samples we are going to to use the QC modules of deepTools (http://deeptools.readthedocs.io/), a software package for the QC, processing and analysis of NGS data. Before computing the correlation a time consuming step is required, which is to compute the read coverage over a large number of regions from each BAM file. For this we will use the tool **multiBamSummary**.